src/Polly.Core/CircuitBreaker/CircuitBreakerStrategyOptions.TResult.cs

```diff
@@ -69,6 +69,14 @@ public class CircuitBreakerStrategyOptions<TResult> : ResilienceStrategyOptions
     public TimeSpan BreakDuration { get; set; } = CircuitBreakerConstants.DefaultBreakDuration;
 #pragma warning restore
 
+    /// <summary>
+    /// Gets or sets the function responsible for dynamically generating the break duration based on the health failure count.
+    /// </summary>
+    /// <value>
+    /// A function that takes an integer representing the health failure count and returns a TimeSpan indicating the break duration.
```

Comment:
I wonder if this taking an `int` boxes us into a corner on whether people might want to use _other_ information to calculate the break duration.

```suggestion
    /// Gets or sets an optional delegate to use to dynamically generate the break duration based on the health failure count.
    /// </summary>
    /// <value>
    /// A delegate that takes an integer representing the health failure count and returns a <see cref="TimeSpan"/> indicating the break duration.
```